src/common/base/src/mem_allocator/jemalloc.rs

```diff
@@ -224,61 +212,3 @@ pub mod linux_or_macos {
         }
     }
 }
-
-#[cfg(not(any(target_os = "linux", target_os = "macos")))]
```

Comment:
If arm works we can remove these codes, we only support Linux and macOS and arm target.